app/views/admin/provisioning_keys/_form.html.haml

```diff
@@ -0,0 +1,13 @@
+= GovukElementsErrorsHelper.error_summary @token,
```

Comment:
The `@token` should be `@provisioning_key` - that's why there's no error summary message at the top of the page.